### PR TITLE
Add documentation to make_f_mo() function

### PR DESCRIPTION
**Pull request by mlestep**

### CODE REVIEW

**Comment by gevahn** [comment]
response calculation functions, not integrated with the rest a.t.m

File: quest/response/response.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import scipy.sparse.linalg as spla
+import scipy as sp
+from hessian_builders import *
```
Comment (dgasmith) [inline quest/response/response.py:4]: `from . hessian_builders import *`

File: quest/response/hessian_builders.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+        FF +=np.einsum('ij,jk,lk->il',Cocc,JK,C)
+    
+        return FF
+    return return_func_pointer
+
+
+
+
+def get_E(F, g, nocc, nbas):
```
Comment (dgasmith) [inline quest/response/hessian_builders.py:71]: How about `build_rhf_orbital_hessian`? Something a bit more explicit.

File: quest/response/hessian_builders.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        V = np.einsum('ij,ik,kl->jl',Cocc,kocc,C,optimize=True) #Eq. (14) in the handout
+        J,K = get_JK(V);
+
+        JK = 4 * J - K - K.T; # denote the centor part of Eq.(16)
+        FF = np.einsum('ij,jk->ik',Focc,kocc) - np.einsum('ij,jk->ik',kocc,F)
+
+        FF +=np.einsum('ij,jk,lk->il',Cocc,JK,C)
+    
+        return FF
+    return return_func_pointer
```
Comment (dgasmith) [inline quest/response/hessian_builders.py:66]: Currying I see.

File: quest/response/__init__.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (dgasmith) [inline quest/response/__init__.py:1]: Dont forget little strings at the top of each file to explain what they are doing.

**Comment by gevahn** [comment]
response (only polarizability for now) to be called with a wfn object)

**Pull request merged**